### ISSUE TITLE
Pseudo dice has positive and negative numbers. Is this a problem?

### ISSUE THREAD

**Issue by drnjld** [MIC-DKFZ/nnUNet#2369]
2024-07-12 20:38:39.310014: Epoch 812
2024-07-12 20:38:39.310821: Current learning rate: 0.00222
2024-07-12 20:42:14.514886: train_loss -0.7843
2024-07-12 20:42:14.517647: val_loss -0.7418
2024-07-12 20:42:14.518608: Pseudo dice [220.6748, -1000.154, 49.0404, -48.9845]
2024-07-12 20:42:14.519396: Epoch time: 215.21 s

2024-07-12 20:42:16.495922: Epoch 813
2024-07-12 20:42:16.496738: Current learning rate: 0.00221
2024-07-12 20:45:52.622278: train_loss -0.7793
2024-07-12 20:45:52.625201: val_loss -0.7747
2024-07-12 20:45:52.626120: Pseudo dice [-94.4669, -72.9265, 39.2226, 67.8997]
2024-07-12 20:45:52.627131: Epoch time: 216.13 s

for Pseudo dice, has positive and negative numbers. Is this a problem?

**Comment by drnjld** [comment]
when it comes to Region-based training, like data in BraTs, how to compute the Pseudo dice?

**Comment by drnjld** [comment]
maybe the seg_to_regions.py has some problems, the old version can avoid the problem

**Comment by aymuos15** [comment]
This may be an error from your dataset.json? It's not able to accept the labels properly?

**Comment by drnjld** [comment]
yes, there are something wrong when taking  seg_to_regions.py, the old version can run normally
